accepted/0003-interactive-audit-resolver.md

```diff
@@ -0,0 +1,92 @@
+# Interactive audit resolver
+
+## Summary
+
+This proposal is for adding means for a human to resolve issues from `npm audit` and interactively make decisions about each issue. Available as `npm audit resolve` command.
+
+Features:
+- Remember the resolutions and allow them to affect `npm audit` behavior.
+- Decision options include 'ignore', 'remind later', 'fix', 'remove'.
+- Allow tracking who resolved an issue and when using git history.
+
+
+## Motivation
+
+At times, running `npm audit fix` won't fix all audit issues. Fixes might not be available or the user might not want to apply some of them, eg. major version updates or build/test dependencies.
+
+It should be possible to make `npm audit` a step in a CI pipeline to go red every time a new vulnerability affects the project. Managing security of dependencies should be quick to update, effective, easy to audit over time and secure in itself.
+
+For that, the user needs a new tool which makes addressing issues one by one convenient even across a whole ecosystem of projects.
+
+## Detailed Explanation
+
+`npm audit resolve` runs audit and iterates over all actions from the output with a prompt for a decision on each. 
+If fix is available, it's offered as one of the options to proceed. Other options include: ignore, remind later and delete.
+
+All decisions are stored in `audit-resolv.json` file as key-value, where key is `${dependency path}|${advisory id}` and value is a structure specific to the resolution.
```

Comment:
Why resolv instead of resolve?

Reply:
I don't have a strong opinion about this. Wrote it the way it is after some well known files like resolv.conf and didn't give it a second thought. Didn't have to choose between 'resolve' or 'resolved' or 'resolver' but now that I see you ask, I think `audit-resolve.json` is probably a better name.